eZ/Publish/SPI/Persistence/Content/Handler.php

```diff
@@ -51,21 +51,21 @@ public function createDraftFromVersion($contentId, $srcVersion, $userId);
     /**
      * Returns the raw data of a content object identified by $id, in a struct.
      *
-     * A version to load must be specified. If you want to load the current
-     * version of a content object use SearchHandler::findSingle() with the
-     * ContentId criterion.
+     * If you want to load current version, $version number can be omitted to make sure
+     * you don't need to rely on search index (async) or having to load in two steps
+     * (first content info then content, risking changes in between to current version).
      *
      * Optionally a translation filter may be specified. If specified only the
      * translations with the listed language codes will be retrieved. If not,
      * all translations will be retrieved.
      *
      * @param int|string $id
-     * @param int|string $version
+     * @param int|string|null $version
```

Comment:
This gets crazier by the minute :stuck_out_tongue_winking_eye: 
```suggestion
     * @param int|null $version
```